src/main/java/com/example/loosecoupling/dto/StudentDtoForParent.java

```diff
@@ -0,0 +1,15 @@
+package com.example.loosecoupling.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class StudentDtoForParent{
```

Comment:
Instead of using for we can name it as 
ParentServStudentDto
This sounds intuitive and less verbose

Reply:
yes